### DISCUSSION TITLE
Any file as a GitHub Snippet

### DISCUSSION THREAD

**Discussion by GeorgePearse** [community/community#32879] [Other Feature Feedback, Questions, & Ideas]
I write medium and dev.to posts. For both you get the best rendering via GitHub snippets, but really I want to just be able to link to each of my mini files in the demo project. At the minute I'll often create a snippet, which is a copy of a file in repo, and then edit the file, and have to remake the snippet etc. 

This would be so much simpler and nicer if you could use any file as a snippet.

**Comment by jfullstackdev** [comment]
did you try GitHub Gists ? it can embed through JavaScript

**Reply by GeorgePearse** [reply]
Yes but I want to be able to use any GitHub file as a gist is my point

**Reply by GeorgePearse** [reply]
Would make a lot more sense to me

**Comment by jfullstackdev** [comment]
if you're willing to do GitHub API, then maybe you can do that

check this out 

https://docs.github.com/en/rest/repos/contents
